config/manifests/bases/oadp-operator.clusterserviceversion.yaml

```
@@ -33,71 +33,276 @@ spec:
       displayName: DataProtectionApplication
       kind: DataProtectionApplication
       name: dataprotectionapplications.oadp.openshift.io
+      statusDescriptors:
+      - description: Conditions defines the observed state of Velero
```

If this were generated from comments then might need to edit [oadp types](https://github.com/openshift/oadp-operator/blob/bbf19c5b48da7f326da38c4bb50aca74d49a4f96/api/v1alpha1/oadp_types.go#L181) such that references to velero is replaced.
```suggestion
      - description: Conditions defines the observed state of DataProtectionApplication
```

---

@kaovilai Fixed